tests/lib/assertions.py

```diff
@@ -159,19 +159,34 @@ def assert_200_valid_message_id_response_body(resp, message_id, url):
         assert actual == expected
 
     @staticmethod
-    def assert_message_delivered_gov_uk(response, message_id, type):
+    def assert_email_gov_uk(response, message_id):
         for i in range(len(response)):
             if message_id in response[i].get("reference"):
                 assert response[i].get("status") == "delivered"
-                assert response[i].get("type") == type
+                assert response[i].get("type") == "email"
                 assert response[i].get("body") is not None
-                if type == "email":
-                    assert response[i].get("subject") is not None
-                    assert response[i].get("email_address") is not None
-                if type == "sms":
-                    assert response[i].get("phone_number") is not None
+                assert response[i].get("subject") is not None
+                assert response[i].get("email_address") is not None
```

Comment:
Could also drop a break on the end of here (and the one for letter) to save it from iterating through everything, I'm not sure how large this response could potentially be or if it's a problem if we don't.